script.js

```diff
@@ -1,11 +1,35 @@
 let navToggle = document.getElementsByClassName("navToggle")[0]
 let navLinks = document.getElementsByClassName("navLinks")[0]
-
+let filterSelection = document.getElementsByClassName("filterSelection")
+let allImages = document.getElementsByClassName('all')
+// toggle menu
 navToggle.addEventListener("click", menuDown);
-function menuDown(){
-    if(navLinks.style.display === "flex"){
-        navLinks.style.display="none"
-    }else{
-        navLinks.style.display="flex"
+function menuDown() {
+    if (navLinks.style.display === "flex") {
+        navLinks.style.display = "none"
+    } else {
+        navLinks.style.display = "flex"
     }
```

Comment:
This `if else` can be shortened using ternary operator

`navLinks.style.display = navLinks.style.display === "flex" ? "none" : "flex"`